README.md

```diff
@@ -201,17 +201,18 @@ but you can also rebuild them manually if you need :
 ### Adding new features in CSG.js or other modules:
 Since OpenJSCAD is made up of multiple dependent modules (csg.js, openscad-openjscad-translator etc),
 the easiest method is to use ```npm link``` to have a 'live' updating development version of OpenJSCAD:
-
-For example for CSG.js
 - create a base directory
-- clone this repository ```git clone git@github.com:jscad/OpenJSCAD.org.git```
+- clone this repository ```git clone https://github.com/jscad/OpenJSCAD.org.git```
 - go into OpenJSCAD.org folder ```cd OpenJSCAD.org```
 - install dependencies ```npm install```
-- start dev server : ```npm run start-dev```
+- start dev server if desired: ```npm run start-dev```
+- make it so global ```openjscad``` refers to local code: ```npm link```
```

Comment:
I think this can be rephrased a bit : something like 
if desired, make the ```openjscad```command refer to the code in this folder

Reply:
I like that.  That also makes it clear that the `npm link` isn't a prereq for the `npm run start-dev`, even if I reverse the order of the two steps.   So I reversed them, which I think is the more natural order.